### ISSUE TITLE
Provide a way to kill a task or terminal

### ISSUE THREAD

**Issue by bpasero** [Microsoft/vscode#56838]
Now that both "Terminal" and "Tasks" are under the same menu (via https://github.com/Microsoft/vscode/issues/56836), @Tyriar suggested to somehow preserve an action in the menu to kill a terminal. Since we already have "Terminate Task..." which in most cases is just a terminal, one idea would be to show a combined picker of tasks and terminals to kill.

**Comment by bpasero** [comment]
@dbaeumer yeah, now that the menu is called "Terminal" I wonder if some of the entries should just show a list of terminals:
* Terminate Task => Kill Terminal
* Show Running  Tasks => Show Terminals
* Restart Running Task => Restart Task in Terminal

Each of these would open a picker with all terminals, and maybe the ones with a task should be grouped together.

**Comment by dbaeumer** [comment]
+1

**Comment by bpasero** [comment]
@dbaeumer this would mean we would probably leave out people that run tasks through output channel still, is that being used a lot still?

**Comment by Tyriar** [comment]
But we would be leaving out people specifically for the menu only, all the old commands will still be accessible via the command palette?

**Comment by bpasero** [comment]
@Tyriar can you explain what you mean by "But we would be leaving out people specifically for the menu only"?

**Comment by dbaeumer** [comment]
I think it would be fair to as old task users to use the command palette. I can look into telemetry to see how many `0.1.0` users exists.

**Comment by Tyriar** [comment]
@bpasero I clarified in the other half of the sentence, perhaps a `;` would have been more appropriate than a `,`. We're not cutting those users off all together, they can still go via command palette.

**Comment by bpasero** [comment]
@Tyriar yeah not sure, I would argue we should not support the legacy task system at all and make every command target the tasks in the terminal. Maybe that requires some extra code to ping users to transition if they have not yet.
